src/Console/Omnia.CLI.csproj

```diff
@@ -19,6 +19,7 @@
   <ItemGroup>
     <PackageReference Include="CsvHelper" Version="15.0.5" />
     <PackageReference Include="DotNetCore.NPOI" Version="1.2.2" />
+    <PackageReference Include="Esprima" Version="2.0.0-beta-1328" />
```

Comment:
Exists some reason to use beta version?

Reply:
When using latest stable release, javascript file is not processed correctly, returning "Object reference" errors on Multiline comment evaluation